packages/orchestration/src/exos/exo-interfaces.ts

```diff
@@ -1,8 +1,8 @@
 import type { IBCConnectionID } from '@agoric/vats';
 import type { Vow } from '@agoric/vow';
 import type { IcaAccount } from '../cosmos-api.ts';
-import type { ICAChannelAddressOpts } from '../utils/address';
-import type { ICQConnection } from './icq-connection-kit';
+import type { ICAChannelAddressOpts } from '../utils/address.js';
+import type { ICQConnection } from './icq-connection-kit.js';
```

Comment:
FWIW, we should be consistent in the extension used. Above is using `.ts`, which I'm ok with since it's an `import type`, but I do believe it's a risk for all of them since runtime imports would require a `.js` extension.

Consider the following:
```suggestion
import type { IcaAccount } from '../cosmos-api.js';
import type { ICAChannelAddressOpts } from '../utils/address.js';
import type { ICQConnection } from './icq-connection-kit.js';
```

I am confused however why this didn't result in a check failure since from what I gather, the plugin requires `.js` extensions to be used.

Reply:
I tried https://github.com/eslint-community/eslint-plugin-n/blob/master/docs/rules/file-extension-in-import.md and it has the same problem.

Maybe by the time the lint rule sees the AST it's `.js`? Dunno.

Btw I tried out using the `eslint-plugin-n` rule but it balks at valid subpath imports like,
```js
import { sha256 } from '@noble/hashes/sha256';
import { bytesToHex } from '@noble/hashes/utils';
```